index.md

```diff
@@ -1,52 +1,415 @@
-<!-- TOC -->
-
 - [Tools](#tools)
-  - [tfwrapper](#tfwrapper)
+  - [docker-terraform-python](#docker-terraform-python)
+  - [jinjaform](#jinjaform)
+  - [python-terrafile](#python-terrafile)
+  - [terraform-datadog-scripts](#terraform-datadog-scripts)
+  - [terraform-wrapper](#terraform-wrapper)
 - [Providers](#providers)
-  - [Cloud-Deploy](#cloud-deploy)
-  - [Gitlab](#gitlab)
-  - [Azure DevOps](#azure-devops)
```

Comment:
Those providers seem to be missing in the new version.

Reply:
+1